.travis.yml

```diff
@@ -8,4 +8,4 @@ before_install:
 
 script:
   - swiftlint
-  - set -o pipefail && xcodebuild -workspace KataScreenshot.xcworkspace -scheme 'KataScreenshot' -destination 'platform=iOS Simulator,name=iPhone 6s Plus' build test CODE_SIGN_IDENTITY=- | xcpretty -c
```

Comment:
You switched form an 6s Plus to 8, so screenshots don't match.
You should have switched to an iPhone 8 Plus to update the screenshots